documentation/book/assembly-deployment-configuration.adoc

```diff
@@ -17,6 +17,8 @@ This chapter describes how to configure different aspects of the supported deplo
 * Kafka Connect clusters with _Source2Image_ support
 * Kafka Mirror Maker
 
+include::assembly-custom-resources.adoc[leveloffset=+1]
```

Comment:
I'm not convinced by this placement. The user IMHO opens the book and either:
* Reads from the beginning of the getting started where latest in 2.2 runs into custom resources. This would be for example new user. before he gets here, he will either understand what CRDs are or will be lost and gone.
* The users wants to find some particular details. In that case he skips the getting started and goes to deployment configuration where the first thing he runs into some explanation about CRDs which he probably already knows.

So, wouldn't it be better to place this either into [2.1](https://strimzi.io/docs/master/full.html#downloads-str) or after 2.1?

Reply:
I've placed after 2.1. See what you think.